src/main/java/pl/javastart/bootcamp/utils/YtLinksParser.java

```diff
@@ -0,0 +1,31 @@
+package pl.javastart.bootcamp.utils;
+
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+@Component
+public class YtLinksParser {
+
+    public List<String> parseVideoLinks(String[] videoLinks) {
```

Comment:
Zmień tę metodę na statyczną.
Definiowanie tej metody jako metody instancji, gdy nie operuje ona na stanie obiektu, ani nie wymaga żadnych zewnętrznych zależności, tylko komplikuje sprawę o tyle, że później musimy ten komponent wstrzykiwać do wszystkich serwisów, niepotrzebnie rozbudowując konstruktory.
Jeżeli metoda będzie statyczna, to klasa nie będzie musiała być komponentem Springa